particles/Magenta/schemas/Sequence.arcs

```diff
@@ -0,0 +1,31 @@
+// Basic schema definitions for working with note sequences in magenta-js.
+// Distilled from review of https://hello-magenta.glitch.me/
+
+schema Note
+  // `index` is used to allow storing notes in a Collection with an order,
+  // since we don't yet have support for ordered lists.
+  Number index
+  Number pitch
+  Number quantizedStartStep
+  Number quantizedEndStep
+  Boolean isDrum
+
+schema QuantizationInfo
+  Number stepsPerQuarter
+  Number qpm
+
+schema TempoEntry
+  // `index` is used to allow storing entries in a Collection with an order,
+  // since we don't yet have support for ordered lists.
+  Number index
+  Number time
+  Number qpm
+
+schema Song
```

Comment:
The full spec for notesequence is here: https://github.com/tensorflow/magenta-js/blob/1bf6acc32b41a90987dfc908846cd2b3f0b69242/music/src/protobuf/proto.d.ts#L10

This is good for now, we should see if we can get these schemas integrated before we expand fields.